### PR TITLE
add my script for the task

### PR DESCRIPTION
**Pull request by virginiah894**

### CODE REVIEW

**Comment by virginiah894** [comment]
Add email

File: scripts/04021-virginiahperiah.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 full_name = 'Virginiah Periah'
 hng_num = '04021'
 used_lang = 'Python'
+email = 'virgyperry@gmail.com'
 print(f" Hello World, this is {full_name} with HNGi7 ID {hng_num} using {used_lang} for stage 2 task.")
```
Comment (ifatoki) [inline scripts/04021-virginiahperiah.py]: Hi, you only added the email variable but we'd need you to print it after the word `task` and one space. You can find more info [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)

**Review by ifatoki — DISMISSED** [review]

**Comment by virginiah894** [comment]
Add the email.

**Review by McToluene — CHANGES_REQUESTED** [review]
Please include HNG in your HNG-ID output. Check here for more info https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700

**Comment by virginiah894** [comment]
HI,
Done

**Comment by virginiah894** [comment]
Fix HNg Id

**Comment by virginiah894** [comment]
fixed the HNG ID

**Review by melissaugrai — APPROVED** [review]

**Review by onedebos — DISMISSED** [review]

**Review by onedebos — CHANGES_REQUESTED** [review]
You need to remove the space you have before "Hello". print(f" Hello World, this is {full_name} with HNGi7 ID {hng_num}  using {used_lang} for stage 2 task. {email}") => should be print(f"Hello World, this is {full_name} with HNGi7 ID {hng_num}  using {used_lang} for stage 2 task. {email}")

**Comment by virginiah894** [comment]
i have fixed the space

**Comment by virginiah894** [comment]
new pull request

**Review by Mahalinoro — APPROVED** [review]
Reviewed!

**Review by onedebos — APPROVED** [review]

**Pull request merged**